lib/command/terraform/import.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Command
+  module Terraform
+    class Import < Base
+      SUBCOMMAND_NAME = "terraform"
+      NAME = "import"
+      OPTIONS = [
+        app_option,
+        dir_option
+      ].freeze
+      DESCRIPTION = "Imports terraform resources"
+      LONG_DESCRIPTION = <<~DESC
+        - Imports terraform resources from the generated configuration files.
```

Comment:
Minor suggestion: we generally don't add a period at the end of each item in the description.

```suggestion
        - Imports terraform resources from the generated configuration files
```